<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightMusic</class>
 <widget class="QWidget" name="LightMusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="deviceBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="modeBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>87</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QSlider" name="volumelSlider">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="SoundLevel" name="soundLevel" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorPallete" name="colorPallete" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>491</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose device</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SoundLevel</class>
   <extends>QWidget</extends>
   <header location="global">mywidgets.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPallete</class>
   <extends>QWidget</extends>
   <header location="global">mywidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
